src/Command/Validators.php

```diff
@@ -32,6 +32,8 @@ public static function validateUsername(?string $username): string
 
     /**
      * @throws \InvalidArgumentException
+     *
+     * @phpstan-return class-string
```

Comment:
The parameter of this function is also a class string right?

Reply:
Since we're doing 
```
str_replace('/', '\\', $class);
```
I would say it's only a class string AFTER the transformation and not before.